tests/sparseml/pytorch/sparsification/pruning/test_modifier_pruning_topkast.py

```diff
@@ -125,7 +125,6 @@ def _test_compression_sparsity_applied():
             assert not modifier.update_ready(epoch, test_steps_per_epoch)
             _test_compression_sparsity_applied()
 
-    @pytest.mark.skip(reason="This test is flaky; fix is in progress")
     @flaky(max_runs=3, min_passes=2)
```

Comment:
Nit: now when the tests are potentially not flaky anymore, consider making the "flakiness" more lenient, potentially removing the `@flaky` completely

Reply:
Hi Damian, I considered this. But I left it in, because it seems pretty harmless and the test still relies on randomness (in the network init), so it actually could rarely fail.